src/cli/bin.js

```diff
@@ -7,10 +7,12 @@ require('dnscache')({ enable: true })
 const pkg = require('../../package')
 const path = require('path')
 const os = require('os')
+const rc = require('rc')
 
 process.title = pkg.name
 
 const yargs = require('yargs')
+  .config(rc('ipfsnpm', null, {}))
```

Comment:
Please use `pkg.name` instead of hard coding it.  nb this will make it`ipfs-npm` and not `ipfsnpm`.